invenio_app_rdm/records_ui/views/deposits.py

```diff
@@ -24,11 +25,53 @@
 #
 # Helpers
 #
+
+def get_form_pids_config():
+    """Prepare configuration for the pids field."""
+    service = current_rdm_records.records_service
+    pids_providers = []
+    for scheme, providers in service.config.pids_providers.items():
+        can_be_managed = False
+        can_be_unmanaged = False
+        provider_enabled = False
+        for name, provider_attrs in providers.items():
+            is_enabled = provider_attrs.get("enabled", True)
+            if not provider_enabled and is_enabled:
+                provider_enabled = True
+
+            if provider_attrs["system_managed"]:
+                can_be_managed = True
+            else:
+                can_be_unmanaged = True
+
+        # all providers disabled for this scheme
+        if not provider_enabled:
+            continue
+
+        u_scheme = scheme.upper()
```

Comment:
minor: This is good for now, but I think there might be cases when is not full upper case (e.g. `PubMed`).

Reply:
True... in any case this requires some more work to override such config...